### DISCUSSION TITLE
운영 체제

### DISCUSSION THREAD

**Discussion by zini9188** [Shortudy-10th/tech-for-developer#1] [운영 방식]
소주제 추천받습니다.

**Comment by watty-exclamationMark** [comment]
프로세스와 스레드 (프로세스와 스레드의 차이)
프로세스 임계영역과 해결책

**Comment by sikutisa** [comment]
* 프로세스 스케줄링과 컨텍스트 스위칭
* Blocking & Non-Blocking vs Sync & Async
